ViteMaDose/Resources/Localization/Localization.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  Localization.swift
+//  ViteMaDose
+//
+//  Created by Victor Sarda on 24/04/2021.
+//
+// swiftlint:disable identifier_name
+
+import Foundation
+
+enum Localization {
+    enum Home {
+        static let select_county = "home.select_county".localized
+        static let recent_search = "home.recent_search".localized
+        static let last_stats = "home.last_stats".localized
+        static let open_map = "home.open_map".localized
+        static let partners = "home.partners".localized
+
+        enum MainTitle {
+            static let title = "home.main_title".localized
+            static let first_highlighted_text = "home.main_title.first_highlighted_text".localized
+            static let second_highlighted_text = "home.main_title.second_highlighted_text".localized
+        }
+
+        enum Stats {
+            static let all_locations = "home.stats.all_locations".localized
+            static let all_availabilities = "home.stats.all_availabilities".localized
+            static let locations_with_availabilities = "home.stats.locations_with_availabilities".localized
+            static let available_locations_percentage = "home.stats.available_locations_percentage".localized
+        }
+    }
+
+    enum CountySelection {
+        enum MainTitle {
+            static let title = "county_selection.main_title".localized
+            static let highlighted_text = "county_selection.main_title.highlighted_text".localized
+        }
+    }
+
+    enum LocationsList {
+        static let list_title = "locations.list_title".localized
+        static let no_results = "locations.no_results".localized
```

Comment:
Misleading names between the Localizable IDs and the enums. `Locations` is not formally tied to `LocationsList`. It would be preferable to create a dedicated `Locations` enum.
```suggestion
    enum Locations {
        static let list_title = "locations.list_title".localized
        static let no_results = "locations.no_results".localized
    }

    enum LocationsList {
```

Reply:
Nice catch